### DISCUSSION TITLE
installation on mac

### DISCUSSION THREAD

**Discussion by jawelo** [ccbogel/QualCoder#816] [Q&A]
Hi!
I would really love to use QualCoder for my master thesis but I'm having trouble with the installation instructions as I'm not experienced with programming. Can anybody give me a hint, if there is an easier understandable instruction than the one here on GitHub?
Thanks so much!

**Comment by ccbogel** [comment]
Have you seen this video for install on macOS?
 https://temple.hosted.panopto.com/Panopto/Pages/Embed.aspx?id=db149143-5c05-41d6-9b6c-b02e01181171

Try this first, then maybe, just maybe, if you are still having troubles we might be able to Zoom to help you. Although I am in Australian Eastern Standard Time Zone and would Zoom during my daytime.

**Reply by jawelo** [reply]
Thanks, I haven't seen that one! Still things are different on my computer than in the video.
I'll ask a friend for help tomorrow and if nothing helps I'll come back to your offer. Thank you!

**Reply by Cloudness24** [reply]
Hi, I had a problem installing it on my Mac. I followed the video and the instructions. However, it says: 
error: metadata-generation-failed.
I don't have any clue what that means; could you please help me?

**Reply by ccbogel** [reply]
I am not sure I can help. I don't know macOS well, is it a modern macOS?. Was there any more information than that?

**Reply by eurostile31** [reply]
Hello Cloudness24, I had the same issue when installing QualCoder yesterday. Did you install Python 3.12 on your computer? Because one of the library (wordcloud) is not yet compatible with Python 3.12 which causes the error you describe. I just installed Python 3.11 and could install and use QualCoder flawlessly.
